CMakeLists.txt

```diff
@@ -8,6 +8,10 @@ find_package(catkin REQUIRED COMPONENTS
   geometry_msgs
   genmsg
   sensor_msgs
+  vision_msgs
```

Comment:
Those changes should also be reflected in the package.xml
It would also make sense to point to the package.xml instead of the CMakeLists.txt in the Dependencies section of the README.md

Reply:
Fixed.
Yes linking the package.xml would make more sense but xml is ugly to look at, so I guess just link both?